build/common/installer/scripts/tomlparser.rb

```diff
@@ -285,6 +300,10 @@ def get_command_windows(env_variable_name, env_variable_value)
     file.write(commands)
     commands = get_command_windows('AZMON_ADX_DATABASE_NAME', @adxDatabaseName)
     file.write(commands)
+    commands = get_command_windows('AZMON_ENABLE_LOG_LOSS_TRACKING', @disableLogLossTracking)
```

Comment:
log loss detection logic only scoped to linux, right? if yes, do we need these environment variables for windows?

Reply:
I removed the env vars for windows, there should now be no Windows changes.